nntrainer/utils/base_properties.h

```diff
@@ -252,13 +310,13 @@ template <> class Property<std::string> {
  *
  * struct custom_tag: int_prop_tag {};
  *
- * using tag_type = tag_cast<custom_tag, vector_prop_tag>::type
- * static_assert(<std::is_save_v<tag_type, custom_tag> == false);
+ * using tag_type = tag_cast<custom_tag, double_prop_tag>::type
```

Comment:
this is unrelated to this PR, but later, let's update double to float.

Reply:
updated